### ISSUE TITLE
useScrollPosition.defaultProps does nothing

### ISSUE THREAD

**Issue by ckknight** [n8tb1t/use-scroll-position#32]
I see that `useScrollPosition.defaultProps` is defined for some reason. It's a hook function and not a React component, so attaching a `defaultProps` property to the function will not accomplish anything and is just extra noise.

**Comment by n8tb1t** [comment]
You are right! It's just a part of some old implementation, we need to remove it!

**Comment by n8tb1t** [comment]
will be removed in the next version
